client/src/components/game/gui/meters/Meters.jsx

```diff
@@ -123,6 +150,22 @@ function Meters() {
                     />
                 </div>
             </div>
+            <div className="combat-indicator-icon">
+                <img
+                  className="gui-icon"
+                  src={swordIcon}
+                  draggable={false}
+                  onMouseEnter={() => {
+                      GUIState.setTooltipContent(
+                          Tooltip(Utils.getTextDef("Combat tooltip")),
+                      );
+                  }}
+                  onMouseLeave={() => {
+                      GUIState.setTooltipContent(null);
+                  }}
+                />
+                {combatTimer}
```

Comment:
Can be contained in a <span>, also can have the same style applied as the other GUI number counters.
`<span className="high-contrast-text">{combatTimer}</span>`

Reply:
Can also not render it when there is no time left on the timer.
`{ (combatTimer <= 0) && <span className="high-contrast-text">{combatTimer}</span> }`